x/circuit/keeper/msg_server.go

```diff
@@ -133,30 +151,34 @@ func (srv msgServer) TripCircuitBreaker(goCtx context.Context, msg *types.MsgTri
 
 // ResetCircuitBreaker resumes processing of Msg's in the state machine that
 // have been been paused using TripCircuitBreaker.
-func (srv msgServer) ResetCircuitBreaker(goCtx context.Context, msg *types.MsgResetCircuitBreaker) (*types.MsgResetCircuitBreakerResponse, error) {
-	ctx := sdk.UnwrapSDKContext(goCtx)
+func (srv msgServer) ResetCircuitBreaker(ctx context.Context, msg *types.MsgResetCircuitBreaker) (*types.MsgResetCircuitBreakerResponse, error) {
 	keeper := srv.Keeper
-
 	address, err := srv.addressCodec.StringToBytes(msg.Authority)
 	if err != nil {
 		return nil, err
 	}
 
 	// Get the permissions for the account specified in the msg.Authority field
-	perms, err := keeper.GetPermissions(ctx, address)
-	if err != nil {
+	perms, err := keeper.Permissions.Get(ctx, address)
+	if err != nil && !errorsmod.IsOf(err, collections.ErrNotFound) {
```

Comment:
Here too btw.

Reply:
Same as above, but I'll remove the text in the return error so it just returns the error